gui/symfony-docker/templates/pro/equarrisseur/equarrir.html.twig

```diff
@@ -3,10 +3,10 @@
 {% block body %}
     <div class="container-fluid">
         <h1>Équarrir la bête :</h1>
-        <p>Remplissez les champs suivants pour équarrir une bête.</p>
+        <p>Valider pour équarrir la bête définitivement</p>
```

Comment:
this line need to be at the bottom, just before de final button

Reply:
i think we need to talk about this point in voc.